dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderAPI.java

```diff
@@ -29,6 +29,8 @@
 
 	public static final String SYSTEM_FOLDER = "SYSTEM_FOLDER";
 	public static final String SYSTEM_FOLDER_ID = "bc9a1d37-dd2d-4d49-a29d-0c9be740bfaf";
+	public static final String SYSTEM_FOLDER_PATH = "/System folder";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)